codingTest_kjy/나머지구하기.js

```diff
@@ -0,0 +1,5 @@
+function solution(num1, num2) { //정수 num1, num2가 매개변수로 주어질 때, num1를 num2로 나눈 나머지를 return 하도록 solution 함수를 완성해주세요
+    var answer = num1 % num2 ;  // 말그대로 나눈 나머지를 리턴값을 그대로 코드를 만들었습니다.
```

Comment:
```suggestion
    const answer = num1 % num2 ;  // 말그대로 나눈 나머지를 리턴값을 그대로 코드를 만들었습니다.
```
혹은
```suggestion
    return num1 % num2 ;  // 말그대로 나눈 나머지를 리턴값을 그대로 코드를 만들었습니다.
```
이렇게 바로 해주는게 좋을 것 같아요.
코드는 간결 해지면 더 좋고 굳이 안해도 되는 변수의 선언은 줄이는게 좋습니다.